### PR TITLE
db: Fix JSON db errors on tab switch.

### PR DESCRIPTION
**Pull request by kanishk98**
---

**What's this PR do?**

Fixes errors with the `settings.json` file related to problem found [here](https://chat.zulip.org/#narrow/pm-with/53,4241,4922,8973-group/near/756485). 

**Any background context you want to provide?**

We'd ideally like to see a more complete solution for this later on, this just prevents errors in a very frequent situation. 

**Screenshots?**

**You have tested this PR on:**
  - [ ] Windows
  - [x] Linux/Ubuntu
  - [ ] macOS

### CODE REVIEW

**Comment by priyank-p** [comment]
LGTM!

File: app/renderer/js/main.js [hunk 1]
```diff
@@ -410,7 +410,7 @@ class ServerManagerView {
 		// Open all the tabs in background, also activate the tab based on the index
 		this.activateTab(index);
 		// Save last active tab
```
Comment (vsvipul) [inline app/renderer/js/main.js]: A comment explaining why this was required in another process would be good ! LGTM.

Comment (kanishk98) [reply app/renderer/js/main.js]: Good idea, thanks. We can remove the comment once the db problems are fixed.

**Comment by akashnimare** [comment]
Merged. Thanks, everyone.

**Pull request merged**